starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,71 @@
+div{
+    border:1px red solid;
+    
+}
+
+body {
+    font-family: Helvetica;
+}
+
+img {
```

Comment:
Avoid using element selectors (img, div, p, h1...), better use classes and ids. Furthermore, in this case you don't want all the images to be 100% width. The icons are huge!